src/components/movies.js

```diff
@@ -0,0 +1,77 @@
+import { useState, useCallback, useEffect } from 'react';
+import axios from 'axios';
+
+import './movies.css';
+import MovieList from './MovieList';
+import FilterArea from './FilterArea';
+import Header from './header';
+
+const Movies = () => {
+  const [showMovies, setshowMovies] = useState([]);
+  const [ filteredlanguage, setfilteredlanguage] = useState('Select Language..');
+  useEffect(() => {
+    axios.get(`
+      https://api.themoviedb.org/3/movie/popular?api_key=ed4f70d18fa8cbc03fb59ce04847e1bc`)
```

Comment:
same API key is being used in multiple components, store it in a different file as constant and import where required